tensorflow_model_optimization/python/core/sparsity_tf2/BUILD

```diff
@@ -56,6 +56,17 @@ py_library(
     ],
 )
 
+py_library(
+    name = "sparse_utils",
+    srcs = ["sparse_utils.py"],
+    srcs_version = "PY2AND3",
+    visibility = ["//visibility:public"],
+    deps = [
+        # tensorflow dep1,
+        # python:summary tensorflow dep2,
```

Comment:
I don't think you need this specific dependency, since you're not using tf summaries via a direct import